Source: Internal

Info: This patch fixes the issue of pkcs11_common(NULL) being called
when PKCS11 is not enabled.

Status: This patch has been submitted to upstream, not yet accepted.

diff --git a/src/tpmtool.c b/src/tpmtool.c
index c8862f0..3741219 100644
--- a/src/tpmtool.c
+++ b/src/tpmtool.c
@@ -196,7 +196,9 @@ tpm_test_sign(const char *url, FILE * out)
 	gnutls_datum_t data, sig = {NULL, 0};
 	int pk;
 
+#ifdef ENABLE_PKCS11
 	pkcs11_common(NULL);
+#endif
 
 	data.data = (void*)TEST_DATA;
 	data.size = sizeof(TEST_DATA)-1;
